src/shared.ts

```diff
@@ -42,7 +44,9 @@ export const componentNameMapping: Record<
   "financial-account": "stripe-connect-financial-account",
   "financial-account-transactions":
     "stripe-connect-financial-account-transactions",
-  documents: "stripe-connect-documents"
+  documents: "stripe-connect-documents",
+  "tax-registrations": "stripe-connect-tax-registrations",
```

Comment:
To confirm, there aren't any props on these components yes?

Reply:
Yeah! No props!